modules/oadp-monitoring.adoc

```diff
@@ -0,0 +1,378 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/troubleshooting.adoc
+
+:_content-type: CONCEPT
+[id="oadp-monitoring_{context}"]
+= OADP Monitoring
+
+The {product-title} provides a xref:../../monitoring/monitoring-overview.adoc#about-openshift-monitoring[monitoring stack] that allows users and administrators to effectively monitor and manage their OpenShift clusters, as well as monitor and analyze the workload performance of user applications and services running on the clusters, including receiving alerts if an event occurs.
```

Comment:
Is the `{product-title}` here replaced by "OADP Monitoring" If yes I think it's not precise.

OADP Monitoring does not provide monitoring stack, it *uses* it.

Reply:
The {product-title} is OADP.

Reply:
Oops it's not OADP.

OCP provides a monitoring stack...